politeiad/cmd/politeiatlog/politeiatlog.go

```diff
@@ -0,0 +1,512 @@
+package main
+
+import (
+	"bufio"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/decred/politeia/politeiad/backend"
+	"github.com/decred/politeia/politeiad/backend/tlogbe"
+	"github.com/decred/politeia/politeiad/backend/tlogbe/store"
+	"github.com/decred/politeia/politeiad/plugins/comments"
+	"github.com/decred/politeia/politeiad/plugins/ticketvote"
+	"github.com/decred/politeia/politeiad/sharedconfig"
+	"github.com/google/trillian"
+)
+
+const (
+	defaultTrillianHostUnvetted = "localhost:8090"
+	defaultTrillianHostVetted   = "localhost:8094"
+	defaultTrillianKeyUnvetted  = "unvetted-trillian.key"
+	defaultTrillianKeyVetted    = "vetted-trillian.key"
+)
+
+var (
```

Comment:
What are all of these extra flags?

```
  -alsologtostderr
        log to standard error as well as files
  -blobs
        Print leaf blobs
  -key string
        Encryption key (default "/home/luke/.politeiad/tlogbe.key")
  -log_backtrace_at value
        when logging hits line file:N, emit a stack trace
  -log_dir string
        If non-empty, write log files in this directory
  -logtostderr
        log to standard error instead of files
  -stderrthreshold value
        logs at or above this threshold go to stderr
  -storage_system string
        Storage system to use. One of: [] (default "mysql")
  -testnet
        Use testnet network
  -trillian string
        Trillian database name (vetted/unvetted)
  -v value
        log level for V logs
  -vmodule value
        comma-separated list of pattern=N settings for file-filtered logging
```

Reply:
oops no idea. `go-flags` must be printing its default flags as well or smth. Will fix this